docs/integrations/system_metrics.md

```diff
@@ -1,28 +1,80 @@
-By default, **Logfire** does not collect system metrics.
+The [`logfire.instrument_system_metrics()`][logfire.Logfire.instrument_system_metrics] method can be used to collect system metrics with **Logfire**, such as CPU and memory usage.
 
-To enable metrics, you need to install the `logfire[system-metrics]` extra:
+## Installation
+
+Install `logfire` with the `system-metrics` extra:
 
 {{ install_logfire(extras=['system-metrics']) }}
 
-### Available Metrics
-
-Logfire collects the following system metrics:
-
-* `system.cpu.time`: CPU time spent in different modes.
-* `system.cpu.utilization`: CPU utilization in different modes.
-* `system.memory.usage`: Memory usage.
-* `system.memory.utilization`: Memory utilization in different modes.
-* `system.swap.usage`: Swap usage.
-* `system.swap.utilization`: Swap utilization
-* `system.disk.io`: Disk I/O operations (read/write).
-* `system.disk.operations`: Disk operations (read/write).
-* `system.disk.time`: Disk time (read/write).
-* `system.network.dropped.packets`: Dropped packets (transmit/receive).
-* `system.network.packets`: Packets (transmit/receive).
-* `system.network.errors`: Network errors (transmit/receive).
-* `system.network.io`: Network I/O (transmit/receive).
-* `system.network.connections`: Network connections (family/type).
-* `system.thread_count`: Thread count.
-* `process.runtime.memory`: Process memory usage.
-* `process.runtime.cpu.time`: Process CPU time.
-* `process.runtime.gc_count`: Process garbage collection count.
+## Usage
+
+```py
+import logfire
+
+logfire.configure()
+
+logfire.instrument_system_metrics()
+```
+
+Then in your project, click on 'Dashboards' in the top bar, click 'New Dashboard', and select 'Basic System Metrics (Logfire)' from the dropdown.
+
+## Configuration
+
+By default, `instrument_system_metrics` collects only the metrics it needs to display the 'Basic System Metrics (Logfire)' dashboard. You can choose exactly which metrics to collect and how much data to collect about each metric. The default is equivalent to this:
+
+```py
+logfire.instrument_system_metrics({
+    'system.cpu.simple_utilization': None,  # (1)!
+    'system.memory.utilization': ['available'],
+    'system.swap.utilization': ['used'],
+})
+```
+
+1. `None` simply means that there are no fields to configure for this metric.
+
+To collect lots of detailed data about all available metrics, use `logfire.instrument_system_metrics(base='full')`.
+
+!!! warning
+    The amount of data collected by `base='full'` can be expensive, especially if you have many servers,
+    and this is easy to forget about. Be sure to monitor your usage and costs.
+
+    The most expensive metrics are `system.cpu.utilization/time` which collect data for each core and each mode,
+    and `system.disk.*` which collect data for each disk device. This can result in hundreds of data points per minute.
+    The exact number depends on your machine.
```

Comment:
```suggestion
    and `system.disk.*` which collect data for each disk device. The exact number depends on the machine hardware,
    but this can result in hundreds of data points per minute from each instrumented host.
```